src/pages/card/Tags.tsx

```diff
@@ -0,0 +1,127 @@
+import { useEffect, useState } from "react";
+import { Input, List, Button } from "antd";
+import { IoIosSearch } from "react-icons/io";
+import Strings from "../../utils/localizations/Strings";
+import { useLocation, useNavigate } from "react-router-dom";
+import PageTitleTag from "../../components/PageTitleTag";
+import { useGetCardsMutation } from "../../services/cardService";
+import PaginatedList from "../../components/PaginatedList";
+import InformationPanel from "./components/Tag";
+import { CardInterface } from "../../data/card/card";
+import { UserRoles } from "../../utils/Extensions";
+import { UnauthorizedRoute } from "../../utils/Routes";
+
+interface CardsProps {
+  rol: UserRoles;
+}
+
+const Cards = ({ rol }: CardsProps) => {
```

Comment:
should we rename also this to Tags right?

Reply:
Yes. correction made